extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -609,10 +609,11 @@ angular.module('PaperUI.controllers.configuration', [ 'PaperUI.constants' ]).con
             dict.bridgeUID = thing.bridgeUID
             update = true;
         }
-        if (update) {
+        originalThing.configuration = thing.configuration = {};
+        if (JSON.stringify(originalThing) !== JSON.stringify(thing)) {
             thingService.update({
                 thingUID : thing.UID
-            }, dict);
+            }, thing);
```

Comment:
You are right, no configuration is lost here. Sending empty configurations would just keep the old configurations as they are. In this call we are only interested in updating the thing properties and not its configurations. I've also removed the unnecessary code above. That should make it more clear.